modules/ROOT/pages/configuring-sso.adoc

```diff
@@ -0,0 +1,440 @@
+|===
+|Field|Value
+|Transformation|ExtractMailPrefix()
+|Parameter 1|Attribute
+|Attribute name|user.mail
+|===
+.. Click *Add*.
+
+. From *Manage claim*, click *Save*.
+. Create single sign-on settings in salesforce:
+.. Select *Single Sign-on* and click *Download* from *Federation Metadata XML* in the *SAML Certificates* section.
+.. Go to *Salesforce > Setup > Single Sign-On Settings* and click *New from Metadata File*.
+.. From the *Metadata File* field, click *Choose File*, select the Federation Metadata XML file, and click *Create*.
+.. Complete the following fields:
++
+[%header,cols="1a,2a"]
+|===
+|Field|Value
+|Name |Enter a name.
+|API Name |Enter a name.
+|Entity ID|Enter the value from the Azure Enterprise application's *Identifier (Entity ID)* field. For Example, `https://<YOUR_DOMAIN>.my.site.com/aeh`.
+|SAML Identity Type|Assertion contains the Federation ID from the User object
+|Just-in-time User Provisioning|Enable User Provisioning Enabled
+|User Provisioning Type |SAML JIT with Apex handler
+|SAML JIT Handler |AEHPortalRegistrationHandler
+|Execute Handler As |Select a system administrator.
+|===
+.. Click *Save*.
+====
+Microsoft Azure OpenID Connect is not supported.
+
+
+== Step 2: Add Salesforce Identity Providers 
+
+After the identity provider is configured for Salesforce, add and enable the identity provider from the API Experience Hub UI. When the identity provider is enabled, users can log in to the portal using this identity provider.
+
+. Go to *API Experience Hub > User management*.
+. From the *User management* page, click *Login settings*.
+. From the *Single sign-on (SSO)* section, scroll down to step 2 *Add Salesforce identity providers*.
+. Click *Select identity provider* and select an option from the drop-down menu. 
+. Click *+ Add identity provider*.
+. Move the slider to *Enabled*.
+
+[[add-group-mapping]]
+== Step 3: Add Group Mappings 
+When setting up SSO for the portal, your users must have an identity in both Salesforce and Anypoint Platform. SSO users are mapped to teams using their group names. You must map your users to teams using Access Management. API Experience Hub provides an out-of-the-box team called *AEH Portal - ${salesforceOrganizationId}_${salesforceCommunityId}* that is added automatically as a team in Access Management.
+
+Add group mappings by adding the user to the corresponding profile in API Experience Hub:
+
+. Go to *Access Management* > *Teams*.
+. Click *AEH Portal Guests* and click *AEH Portal Members*.
+. Click *External IdP Groups*.
+. From the *Group Name* field, enter *AEH Members* or enter the value of the groups claim.
+. From the *Provider Name* field, select the name of the corresponding Salesforce identity provider.
+. From the *Type* field, select *Member* and click *Add*.
++
+image::aeh-sso-access-mgmt-ext-idp-groups.png[Access Management group mapping]
+. Click *Save Changes*.
++
+The SSO users associated with the group you designated are assigned to the team.
```

Comment:
suggestion: Maybe we can tell the users to configure the group mappings in AEH
Users can do this in the  User groups section.